app/models/volunteer_op.rb

```diff
@@ -55,10 +55,14 @@ def self.contains_title?(text)
     arel_table[:title].matches(text)
   end
 
-  def self.build_by_coordinates
-    local_vol_ops = vol_op_with_coordinates(VolunteerOp.local_only)
-    vol_ops = local_vol_ops + VolunteerOp.remote_only
-    vol_op_by_coordinates = group_by_coordinates(vol_ops)
+  def self.build_by_coordinates(opts=nil)
```

Comment:
Hi guys @tansaku @vardyb, I added optional parameters to this method when we are querying for certain results since we only want to work with those registers, but I think also, that this it may be a good opportunity to check the logic that it was being implemented, since as I see it:
```
local_vol_ops = vol_op_with_coordinates(VolunteerOp.local_only)
vol_ops = local_vol_ops + VolunteerOp.remote_only
vol_op_by_coordinates = group_by_coordinates(vol_ops)
```
`vol_ops` is gonna have all the values of the table, what do you think?